sdk/servicebus/Azure.ResourceManager.ServiceBus/src/autorest.md

```diff
@@ -11,6 +11,11 @@ clear-output-folder: true
 skip-csproj: true
 modelerfour:
     lenient-model-deduplication: true
+
+override-operation-name:
+    Namespaces_CheckNameAvailability: CheckServiceBusNameAvailability
```

Comment:
Can we align this method name and the resource `ServiceBusNamespace`?
```suggestion
    Namespaces_CheckNameAvailability: CheckServiceBusNamespaceNameAvailability
```

Reply:
After discussion, we thought maybe it's still better to use `NameAvailability`.